test/nl/surf/eduhub_rio_mapper/smoke_test.clj

```diff
@@ -222,6 +222,22 @@
                   :opleidingeenheidcode "1010O8815"}
                  (:dry-run result))))))
 
+    (testing "education-specifications - dry run should have diff false everywhere (but didn't in client test)"
```

Comment:
I still don't understand what tested here.  What client test?  If you're referring to some conversation outside of this codebase, please just make it `(testing "education-specifications" ..` because it's not some special case.  If it is some special case, describe it.

Reply:
To clarify, a dry-run failed for a client, they included the http-interactions, and I turned them into VCR cassettes in order to reproduce the bug.